providers/ms365/resources/ms365.lr

```diff
@@ -28,20 +30,30 @@ microsoft {
   tenantDomainName() string
 }
 
-// Microsoft organization
-private microsoft.organization @defaults("displayName") {
+//  Microsoft Entra tenant
+microsoft.tenant @defaults("name") {
   // Organization ID
   id string
-  // Organization assigned plans
+  // Service plans associated with the tenant
   assignedPlans []dict
-  // Organization creation date
+  // Provisioned plan
+  provisionedPlans []dict
+  // Deprecated: use `createdAt` instead
   createdDateTime time
-  // Organization display name
+  // Deprecated: use `name` instead
   displayName string
+  // Tenant display name
+  name string
   // Organization verified domains
   verifiedDomains []dict
   // Whether password hash sync is enabled for hybrid deployments
   onPremisesSyncEnabled bool
+  // Tenant creation date
+  createdAt time
+  // Tenant type
+  type string
+  // Commercial subscription
+  subscriptions() []dict
```

Comment:
not scope for this PR but maybe w can think of tying this with `azure.subscription`, if possible

Reply:
That is not the same as an azure subscription. Completely different thing :-)